code/Data exploration.R

```diff
@@ -66,6 +98,13 @@ data <- data %>%mutate(SEASON = case_when(
   MONTH %in%  c(12, 1, 2)  ~ "SUMMER",
   MONTH %in%  3:5  ~ "AUTUMN",
   TRUE ~ "WINTER"));
+
+##Season indicator
+data$SPRING <- if_else(data$SEASON == 'SPRING',1,0)
```

Comment:
for consistency, would be best to agree on one form for dummy variables. Here you use 1 and 0 but below you use TRUE and FALSE

Reply:
will change all to {1,0} for consistency 👍